Chat_Log/Message.cs

```diff
@@ -1,15 +1,15 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
 
-namespace ClassLibrary8
+namespace Chat_Log
 {
     public class MessagesQueue
     {
         private Queue<Message> messages = new Queue<Message>();
 
         public void AddMessage(Message message)
         {
-            if (messages.Count >= 300)
+            if (messages.Count >= Core.Instance.Settings.MaxLogMessages)

```

Comment:
Instead of using a hardcoded value it will now use the settings value which can be changed by the server owner